examples/templates/upload.html

```diff
@@ -0,0 +1,22 @@
+<h1> Flask-Reuploaded. Example:1</h1>
+
+<form method=POST enctype=multipart/form-data action="{{ url_for('upload') }}">
+
+    <input type="file" name="photo">
+
+    <input type="submit" value="submit">
+
+</form>{% if url_by_filename %}
+<p>Click this <a href="{{url_by_filename}}">Link</a> to see the last uploaded file.</p>
+<br>
+<small>N.B;. This link generated by the setname & filename</small>
+
+{% endif %}
+<br>
+<br>
+{% if url_by_id %}
+<p>Click this <a href="{{url_by_id}}">Link</a> to see the last uploaded file.</p>
+<br>
+<small>N.B;. This link generated by the file id (server generated)</small>
+
+{% endif %}
```

Comment:
Please add a line break at the end.

I'd also prefer a complete HTML page, see https://github.com/jugmac00/flask-reuploaded#html-code-for-uploadhtml